packages-ts/gauntlet-terra-contracts/src/commands/contracts/ocr2/inspection/responses.ts

```diff
@@ -0,0 +1,195 @@
+    }
+  }
+
+  return {
+    transmitters: transmitters.addresses,
+    description,
+    aggregatorAddress: aggregator,
+    latestRoundData: {
+      roundId: latestRoundData.round_id,
+      answer: latestRoundData.answer,
+      observationsTimestamp: latestRoundData.observations_timestamp,
+      transmissionTimestamp: latestRoundData.transmission_timestamp,
+    },
+    event: event,
+  }
+}
+
+const inspect = (inputData: CommandInput, onchainData: ContractExpectedInfo): boolean => {
+  let inspections: inspection.Inspection[] = [
+    inspection.makeInspection(onchainData.transmitters, inputData.transmitters, 'Transmitters'),
+    inspection.makeInspection(onchainData.description, inputData.description, 'Description'),
+  ]
+  inspection.inspect(inspections)
+
+  const event = onchainData.event
+  if (!event) {
+    logger.error(
+      `Could not find NewTransmission event of aggregator ${onchainData.aggregatorAddress} for the latest round).`,
+    )
+    return false
+  }
+  logger.debug(event)
+  logger.debug(onchainData.transmitters)
+
+  const roundTime = new Date(Number(event.observations_timestamp) * 1000)
+  logger.info(`Found NewTransmission event for round ${event.aggregator_round_id} (${roundTime})`)
+
+  let responsiveNodes: { [address: string]: ObserverInfo } = {}
+  let unresponsiveNodes: { [address: string]: ObserverInfo } = {}
+
+  const parsedObserversIndexes = parseObserversByLength(event.observers, event.observations.length)
+
+  parsedObserversIndexes.forEach((val, i) => {
+    const observation = event.observations[i]
+    const nodeAddress = onchainData.transmitters[val]
+    if (nodeAddress in inputData.observersInfo) {
+      responsiveNodes[nodeAddress] = {
+        name: inputData.observersInfo[nodeAddress]['name'],
+        apis: inputData.observersInfo[nodeAddress]['apis'],
+        observation,
+      }
+    } else {
+      logger.error(`The oracle address was not found in the input data: ${nodeAddress}`)
+    }
+  })
+
+  logger.success(`${Object.keys(responsiveNodes).length} out of ${onchainData.transmitters.length} nodes responded:`)
+  logger.log(responsiveNodes)
+
+  // Count missing responses
+  onchainData.transmitters.forEach((address) => {
+    if (!(address in responsiveNodes)) {
+      if (address in inputData.observersInfo) {
+        unresponsiveNodes[address] = {
+          name: inputData.observersInfo[address]['name'],
+          apis: inputData.observersInfo[address]['apis'],
+        }
+      } else {
+        logger.error(`The oracle address was not found in the input data: ${address}`)
+      }
+    }
+  })
+
+  if (Object.keys(unresponsiveNodes).length > 0) {
+    logger.warn(`Found ${Object.keys(unresponsiveNodes).length} unresponsive nodes:`)
+    logger.log(unresponsiveNodes)
+  } else {
+    logger.success(`All ${onchainData.transmitters.length} nodes responded.`)
+  }
+
+  logger.info(`Aggregated answer in round ${event.aggregator_round_id}: ${event.answer}`)
+
+  return false
```

Comment:
Inspect should return is the inspection was successful. If we reached this point, the transmission was ok, so we just need to run the other inspections
```suggestion
  return inspection.inspect(inspections)
```